libs/net/dal/Elasticsearch/TNOElasticClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Extensions.Options;
+using Nest;
+using TNO.Core.Exceptions;
+using TNO.DAL.Config;
+
+namespace TNO.DAL.Elasticsearch
```

Comment:
Moved this to the DAL instead of being in the API.  We use it in the Elastic Migration Tool.